### ISSUE TITLE
How to print and save the loss of KANs during each epoch?

### ISSUE THREAD

**Issue by JinglaiZheng** [KindXiaoming/pykan#272]
How to print and save the loss of KANs during each epoch? I need to plot the training diagram of KANs.

**Comment by kaneyxx** [comment]
If you use `model.train()` for training. You can obtain those losses by using 
```
results = model.train( )
results['train_loss']
results['test_loss']
restuls['reg']
```
Hope this help you!
